extensions/amp-script/amp-script.md

```diff
@@ -142,6 +142,24 @@ AMP.getState(expr) {}
 </script>
 ```
 
+### Exporting functions for use in <amp-list>
+
+You may export functions to act as the data source for an `<amp-list>`.
```

Comment:
"export a function"?

Reply:
I was trying to show that you may export multiple functions, although that may be obvious enough that I don't need to call it out!